.travis.yml

```diff
@@ -67,6 +68,21 @@ matrix:
     python: 3.8
     dist: bionic
 
+  # Build and deploy the aarch64 wheels on tagged releases to master branch
+  - stage: deploy
+    arch: arm64
+    dist: bionic
+    python: 3.8
+    services:
+      - docker
+    if: (branch IS master) AND (tag IS present) AND (repo = python-greenlet/greenlet)
```

Comment:
There's an `invalid condition` error in travis about this line: https://travis-ci.org/github/python-greenlet/greenlet/builds/724234713/config#rccb_greenlet:.travis.yml@cdaa46788b82e1ea.78

Reply:
Ok, let me look at that.  As for the above comment about automatic uploads, you can point it to testpypi to manually check the wheels before doing the upload to the official pypi index.  

As for needing user name and password, you actually do not need to supply that.  In Travis, you can setup a secure variable through the console called TWINE_USERNAME and TWINE_PASSWORD in the settings for the repository.  You set TWINE_USERNAME=__token__ and then paste in your api token to TWINE_PASSWORD.  These are kept encrypted on Travis servers and won't be exposed in the build logs.

I'll try and get an update to this PR today.

Reply:
@snaury , I fixed the invalid condition.  It comes down to this part of the [Travis documentation](https://docs.travis-ci.com/user/environment-variables/):
> TRAVIS_BRANCH:
> for push builds, or builds not triggered by a pull request, this is the name of the branch.
> for builds triggered by a pull request this is the name of the branch targeted by the pull request.
> for builds triggered by a tag, this is the same as the name of the tag (TRAVIS_TAG).
> Note that for tags, git does not store the branch from which a commit was tagged.

So specifying the tag, and the branch was invalid as they are mutually exclusive.

I also pointed twine to test.pypi.org instead in the new commit. I think that gives you the best of both worlds, you can test the wheels install via `pip` and also download them from test.pypi for manual test and upload later to pypi.   You simply need an account in test.pypi and you can create an API token to store with Travis-CI that you can late revoke if needed.